telemetry/util.go

```diff
@@ -24,7 +24,10 @@ import (
 // hashString returns the SHA1 checksum in hex of the string.
 func hashString(text string) string {
 	hash := sha1.New()
-	hash.Write([]byte(text))
+	_, err := hash.Write([]byte(text))
```

Comment:
I think we can add a return value to this function.

Reply:
Return val added and fixed relevant usages a bit.